lib/commands/performance.js

```diff
@@ -0,0 +1,297 @@
+import { errors } from 'appium-base-driver';
+import { sleep } from 'asyncbox';
+import _ from 'lodash';
+
+let commands = {}, helpers = {}, extensions = {};
+
+commands.postDismissAlert = async function () {
+  throw new errors.NotYetImplementedError();
+};
+
+const SUPPORTED_PERFORMANCE_DATA_TYPES = {
+  'cpuinfo': 'the amount of cpu by user and kernel process - cpu information for applications on real devices and simulators',
```

Comment:
For all of these keys (`cpuinfo`, `memoryinfo`, `batteryinfo`, and `networkinfo`) there is no need for quotes. The keys, as long as they are valid in JavaScript, can be bare.